monroe.el

```diff
@@ -541,7 +541,7 @@ as path can be remote location. For remote paths, use absolute path."
     (let ((n (buffer-file-name)))
       (read-file-name "Load file: " nil nil nil
                       (and n (file-name-nondirectory n))))))
-  (let ((full-path (convert-standard-filename (expand-file-name path))))
+  (let ((full-path (tramp-compat-file-local-name (convert-standard-filename (expand-file-name path)))))
```

Comment:
The function in question does not exist on my Emacs, (26.1) nor on my co-worker's who says he's on 27. I think monroe should check to see if the function exists before calling it; currently this errors out every time.